src/lib/targeting.test.ts

```diff
@@ -72,46 +72,82 @@ describe('shouldNotRenderEpic', () => {
 
 describe('shouldThrottle', () => {
     it('returns true if epic was viewed too recently', () => {
-        const config = { maxViewsDays: 90, maxViewsCount: 4, minDaysBetweenViews: 5 };
-        const viewLog = [{ date: new Date('2019-06-11T10:24:00').valueOf(), testId: 'A' }];
-
+        const config = { maxViewsDays: 90, maxViewsCount: 4, minDaysBetweenViews: 2 };
         const now = new Date('2019-06-12T10:24:00');
-        const got = shouldThrottle(viewLog, config, undefined, now);
-        expect(got).toBe(true);
+
+        // Epic viewed just now
+        const viewLog1 = [{ date: new Date('2019-06-12T10:23:59').valueOf(), testId: 'A' }];
+        const got1 = shouldThrottle(viewLog1, config, undefined, now);
+        expect(got1).toBe(true);
+
+        // Epic viewed yesterday
+        const viewLog2 = [{ date: new Date('2019-06-11T10:24:00').valueOf(), testId: 'A' }];
+        const got2 = shouldThrottle(viewLog2, config, undefined, now);
+        expect(got2).toBe(true);
```

Comment:
Thoughts on splitting each of these assertions out into separate `it` blocks? I think the failure messages tend to be easier to follow if we try and limit each test to one assertion (because the it block name can be more specific).

Reply:
I did like the simplicity of the `it` blocks before and thought it'd make sense to group the assertions in that way because they were all edge cases of the same scenario. But I think you're right and we should aim to make the blocks as small as possible and limit the number of assertions as much as we can.